tasks/main.yml

```diff
@@ -54,6 +54,21 @@
   notify:
     - restart keepalived
 
+- name: Configure keepalived extra params
+  lineinfile:
+    line: "{{ item }}"
+    regexp: "^{{ item.split('=')[0] }}"
+    dest: "{{ keepalived_daemon_options_file_path }}"
+    state: present
+  with_items: "{{ keepalived_daemon_default_options_overrides }}" 
+  when:
```

Comment:
as said above, the when is always true, due to the definition of both variables in defaults.